zcash_keys/src/keys.rs

```diff
@@ -552,72 +552,126 @@ impl fmt::Display for AddressGenerationError {
 #[cfg(feature = "std")]
 impl std::error::Error for AddressGenerationError {}
 
+/// An enumeration of the ways in which a receiver may be requested to be present in a generated
+/// [`UnifiedAddress`].
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum ReceiverRequirement {
+    /// A receiver of the associated type is required to be present in the generated
+    /// `[UnifiedAddress`], and if it is not possible to generate a receiver of this type, the
+    /// address generation method should return an error. When calling [`Self::intersect`], this
+    /// variant is stronger than [`ReceiverRequirement::Allow`].
```

Comment:
What does "stronger" mean here?

Reply:
`Omit` is stronger than `Allow`, as it explicitly forbids the presence of a receiver. `Require` is similarly stronger than `Allow`, because an error will be produced if no receiver of the associated type is present. `Require` and `Omit` are incompatible for the purpose of intersection.